src/components/MapWithMarkers/MapWithMarkers.css

```diff
@@ -0,0 +1,14 @@
+.markers-map-container {
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+
+.toggle-maps-btn {
+  position: absolute;
+  z-index: 999;
+}
+
+.popup {
+  pointer-events: none;
+}
```

Comment:
I remember suggesting this, but im not sure if this "pointer-events" is actually doing anything?